spec.md

```diff
@@ -1601,6 +1655,72 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 | Invalid `starting_token` | 10 ABORTED | Indicates that `starting_token` is not valid. | Caller SHOULD start the `ListSnapshots` operation again with an empty `starting_token`. |
 
 
+#### `ControllerExpandVolume`
+
+A Controller plugin MUST implement this RPC call if plugin has `EXPAND_VOLUME` controller capability.
+This RPC allows the CO to expand the size of a volume.
+
+This call can be made by the CO during any time in the lifecycle of the volume after creation if plugin has `EXPANDABLE_ONLINE` capability.
+If plugin has `EXPAND_VOLUME` node capability, then `NodeExpandVolume` MUST be called after `ControllerExpandVolume` has completed.
+
+
```

Comment:
nits: kill one blank line here

Reply:
TODO?

Reply:
I think github has messed up where this commend was posted originally. There used to be 2 blank lines but now there is only one between this section and the next section and IMO one blank line is needed to keep these sections separate.